src/app/inbound.rs

```diff
@@ -38,6 +40,14 @@ impl ShouldStackPerRequest for Endpoint {
     }
 }
 
+impl CanClassify for Endpoint {
+    type Classify = super::classify::Request;
```

Comment:
Nit/TIOLI: personally, I find referring to these types as `super::foo` a little unclear --- I'd prefer if we used the full path for these. Not a big deal though.